### ISSUE TITLE
ngx-translate not Working in IE11

### ISSUE THREAD

**Issue by Abhishekism9450** [ngx-translate/core#1226]
I am trying to build my application with Angular 9.1.11 , 
I have installed @ngx-translate/core and @ngx-translates/http-loader. When running my application it works fine in Chrome and Edge but gives Error in IE. 
Error I got is : SCRIPT1002: Syntax Error
                         Vendor.js { class TranslateLoader} 

Please let me know about a solution to resolve.

**Comment by MatiasDandrea** [comment]
I had the same issue in the same Angular version.
After a downgrade in the the package.json at version:
"@ngx-translate/core": "11.0.1",
"@ngx-translate/http-loader": "4.0.0",

now works fine on IE 11

**Comment by Wintermoose** [comment]
ngx-translate version 13 is meant for Angular10+  (angular10 can automatically downlevel library bundles, so the ES5 bundle meant for internet explorer is missing in the package). If you use Angular9, version 12.x should work fine (it does for us)

**Comment by Abhishekism9450** [comment]
@Wintermoose @MatiasDandrea yes , I was facing a problem in bundles so I used a third party package and that runs perfectly fine for me.

**Comment by NoorMohammed993** [comment]
> now works fine on IE 11
works for me
